index.js

```diff
@@ -61,10 +73,13 @@ const resolver = Consumer.create({
         const caseID = data.createcaseresponse.caseid;
 
         logger.info({ caseID, message: 'Casework submission successful' });
-
+        submitDuplicate({duplicate: false, case_id: caseID, external_id: externalId})
+          .catch(err => logger.log('error', err.message));
```

Comment:
@Alex-Swann  this is to prevent the following issue occurring

```
{"level":"error","message":"Duplicate submission failed: 5xx - error: insert into \"duplicates\" (\"case_id\", \"duplicate\", \"external_id\") values ($1, $2, $3) - relation \"duplicates\" does not exist"}
(node:45535) UnhandledPromiseRejectionWarning: Error: Duplicate Table Error
    at submitDuplicate (/Users/christophermedland/HO/SAS/HOF/icasework-resolver/index.js:56:11)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

Reply:
We are already handling errors in the submitAudit/submitDuplicate methods. We shouldn't do this as if there is a legitimate error, the resolver should be retrying the payload to ensure a proper audit message is sent to RDS otherwise we'll lose the information to temporary logs in the kube pods.

Reply:
Also do we need to be logging non duplicate payload logs to the duplicates table?

Reply:
yeah we don't need to record non duplicates so can remove it from here